### PR TITLE
Fix hidden search results

### PR DESCRIPTION
**Pull request by Saira-A**
This fixes the issue found in #1726 where search results hide underneath the image by adding a z-index to the autocomplete function

### CODE REVIEW

File: src/content-handlers/iiif/modules/uv-searchfooterpanel-module/css/styles.less [hunk 1]
```diff
@@ -117,6 +117,7 @@
           padding: 0;
           max-height: 300px;
           overflow: hidden auto;
+          z-index: 15;
```
Comment (demiankatz) [inline src/content-handlers/iiif/modules/uv-searchfooterpanel-module/css/styles.less]: Thanks, @Saira-A! Just curious where the 15 value comes from. Doing a search through the code, for example, I see some annotations with a z-index of 20. Is it possible there could be conflicts there?

(In any case, I'm happy to approve and merge this if you're confident it's an improvement -- but just want to be sure there aren't other edge cases we're missing, since it's hard to test every scenario).

Comment (Saira-A) [reply src/content-handlers/iiif/modules/uv-searchfooterpanel-module/css/styles.less]: @demiankatz I went with 15 because it was the lowest value that worked, but can increase it to be safe - could do 25 to make sure it stays in front of annotations?

Comment (demiankatz) [reply src/content-handlers/iiif/modules/uv-searchfooterpanel-module/css/styles.less]: Yeah, 25 sounds like it might be worth a try. We'd just want to check the various dialog boxes and make sure everything stacks in a logical way. Let me know if you need help with that part!

Comment (Saira-A) [reply src/content-handlers/iiif/modules/uv-searchfooterpanel-module/css/styles.less]: Thanks @demiankatz - looks ok to me but an extra pair of eyes wouldn't hurt :)

**Review by demiankatz — APPROVED** [review]
Thanks, @Saira-A, I can't find a way to break it. :-)
